_includes/utilities/responsive-variants.html

```diff
@@ -14,7 +14,7 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
 }
 </pre>
 </div>
-  <p class="utility-text margin-top-4">Set which breakpoints are available to USWDS utilities with the <code>$theme_output_breakpoints</code> variable found in <code>_uswds-theme-utilities.scss</code>.</p>
+  <p class="utility-text margin-top-4">Set which breakpoints are available to USWDS utilities with the <code>$theme_output_breakpoints</code> variable found in<code>_uswds-theme.scss</code>.</p> 
```

Comment:
`$theme_output_breakpoints` no longer exists. Also removed "found" from the copy.

```suggestion
  <p class="utility-text margin-top-4">Set which breakpoints are available to USWDS utilities with the <code>$theme-utility-breakpoints</code> variable in <code>_uswds-theme.scss</code>.</p> 
```